ui/src/app/index.js

```diff
@@ -3,7 +3,7 @@
 angular.module('bassa', ['ngAnimate', 'ngCookies', 'ngTouch',
   'ngSanitize', 'ui.router', 'ngMaterial', 'nvd3', 'app'])
 
-  .value('BassaUrl', 'http://localhost:5000')
+  .value('BassaUrl', 'http://localhost/api')
```

Comment:
I think if we change this to just `/api` this will work across all the scenarios even when the host is not `localhost`.